src/app/tabs/staticanalysis/staticAnalysisView.js

```diff
@@ -105,6 +107,9 @@ staticAnalysisView.prototype.run = function () {
   if (!this.view) {
     return
   }
+  const highlightLocation = (location, fileName) => {
+    this.sourceHighlighter.currentSourceLocationFromfileName(location, fileName)
```

Comment:
if you have access to the plugin API from here, it'd be better to use it:
`await remix.call('editor', 'highlight', pos, 'browser/3_Ballot.sol')`
and see https://github.com/ethereum/remix-ide/pull/2815/files#diff-bad7952ccc3083125a4c5549afda1978R133 for example how to set highlight, remove a specific highlight and remove all highlights

Reply:
probably that we should then move the debugger to using this API also

Reply:
#2834